Dockerfile (#44)
coleygroup/pyscreener: README.md

All python dependencies and the pyscreener library are installed to a conda environment named `pyscreener` which must be activated once the docker image starts. 

The below commands can be run in the directory containing the `Dockerfile` and `environment.yml` files to build the desired image:
* `docker build -t pyscreener:base --target base .` : Creates a docker image containing all python dependencies and pyscreener library but no docking software 
* `docker build -t pyscreener:vina --target vina .` : Creates an image from `pyscreener:base` with `vina` installed 
* `docker build -t pyscreener:qvina --target qvina .` : Creates an image from `pyscreener:base` with `qvina` installed 
* `docker build -t pyscreener:smina --target smina .` : Creates an image from `pyscreener:base` with `smina` installed 
* `docker build -t pyscreener:psovina --target psovina .` : Creates an image from `pyscreener:base` with `psovina` installed 

As `DOCK6` software requires a license, it is not possible to include its installation within the associated docker image.

> A compiled form of `sphgen_cpp` and the binary required for installation of `chimera` are both available within the `dock6_utils` directory of the associated dock6 imge:

Reviewer: ```suggestion
A compiled form of `sphgen_cpp` and the binary required for installation of `chimera` are both available within the `dock6_utils` directory of the associated dock6 image:
```

* `docker build -t pyscreener:dock6 --target base-dock6 .` : Creates an image from `pyscreener:base` containing utility software needed for `DOCK6` to run once installed 

> Notes : 

Reviewer: ```suggestion
**Notes :** 
```

> 1. Within the docker container, the environment `base` will e activated by default which contains all the required python dependencies so there is no need to manually activate an environment once inside the container

Reviewer: ```suggestion
1. Within the docker container, the environment `base` will be activated by default. This contains all the required python dependencies so there is no need to manually activate an environment once inside the container
```

> 2. If installing using docker, then the below installation stages are not required with the exception of `DOCK6` however.

Reviewer: ```suggestion
2. If installing using docker, then the below installation stages are not required for the corresponding vina-type software. However, the DOCK6 directions must still be followed.
```
